packages/player/src/components/cue-bar.tsx

```diff
@@ -56,7 +56,6 @@ const NoteCue: React.FC<any> = ({cue, duration, className}) => {
       source: 'cue',
     })
     videoService.send('END_SEEKING')
-    videoService.send('PAUSE')
```

Comment:
dunno if I agree with this in terms of UX where somebody has explicitly clicked on a cue. in practice it's weird when it keeps playing, but I might not be seeing the whole picture in context

Reply:
fixed 👍